config/manager/manager.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: apps/v1
```

Comment:
When I generated 0.19.3 scaffolding in cfc695634ec, it wanted to bring in a:

```yaml
apiVersion: v1
kind: Namespace
metadata:
  labels:
    control-plane: controller-manager
  name: system
```

entry here too.  Any idea what that's about?

Reply:
It's used by `kustomize`. If you look at the original Makefile generated you'll see it's used in recipes `'deploy` and `bundle`. The Namespace piece allows you to proliferate a namespace throughout your manifests as I recall. I dumped most of those recipes for this initial PR, where it's using the existng makefile and hack scripts. The `bundle` recipe comes back when I do the bundle PR but I don't see us using the `deploy` or have the need for the `Namespace` in that file.